src/test/java/com/checkmarx/flow/cucumber/integration/pullrequest/updatecomments/UpdatePullRequestCommentsSteps.java

```diff
@@ -94,20 +111,84 @@ private void initSca() {
         scaProperties.setAccessControlUrl("https://platform.checkmarx.net");
     }
 
+    private class ScanResultsAnswerer implements Answer<ScanResults> {
+
+        private FilterConfiguration filterConfiguration;
+
+        private FilterConfiguration switchFilterConfiguration(){
```

Comment:
better to write:
return filterConfiguration == filterMedium ? filterLow : filterMedium ;

Reply:
Fixed